jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    moduleFileExtensions: ["js", "ts"],
+    preset: "ts-jest",
+    roots: ["<rootDir>/server"],
+    testRegex: "\\.test\\.ts$",
```

Comment:
Is it normal for this regex to only include one of the `moduleFileExtensions`?

Reply:
Here is the documentation for `moduleFileExtensions`:
> An array of file extensions your modules use. If you require modules without specifying a file extension, these are the extensions

It also says:
> if you are using TypeScript, you may want to consider moving "ts" and/or "tsx" to the beginning of the array.

→ I have something to do here :sweat_smile: 

For `testRegex`:
> The pattern or patterns Jest uses to detect test files.

It seems to be quite different.